src/cluster.c

```diff
@@ -2878,6 +2867,8 @@ void clusterSetGossipEntry(clusterMsg *hdr, int i, clusterNode *n) {
 /* Send a PING or PONG packet to the specified node, making sure to add enough
  * gossip information. */
 void clusterSendPing(clusterLink *link, int type) {
+    static unsigned long long call_times = 0;
```

Comment:
perhaps renaming `call_times` to `ping_round` is easier to understand, `last_in_ping_message` -> `last_ping_round` is the same.

Reply:
I agree, `ping_round ` is better, but I was worried that `last_ping_round` would be confusing with `ping_sent`, so I thought renaming it to `last_in_ping_round`, so ok?

Reply:
i'd suggest `cluster_pings_sent` (named after the function name), or simply `sent_counter` or `iteration_counter`.
i don't like the "time" part of "times", so "counter" or "ping**s**_sent" solves this.

Reply:
done